notebooks/Coco.ipynb

```diff
@@ -18,14 +18,16 @@
     "from pyspark.sql import SparkSession\n",
     "import pyspark.sql.functions as F\n",
     "import numpy as np\n",
+    "from rikai.spark.utils import get_default_jar_version\n",
     "\n",
+    "version = get_default_jar_version(False)\n",
```

Comment:
should we use `get_dfeault_jar_version(use_snapshot=False)` to make it more readable?

Reply:
if we dont use snapshot version in notebook, will it already fail?
For example, it is `0.0.4-snapshot` at the moment, this notebook will try to include `0.0.4` which has not been released?

Reply:
it will fail

Reply:
doesnt that mean if anyone open a notebook from master branch, the notebook will not work?

Reply:
yeah, let me update this. I broke this out from the sbt-ci-release change but i forgot i manually bumped the dynver patch version number :(

Reply:
i think just setting `use_snapshot=True` would fix the problem